internal/btpmanager/credentials/manager_test.go

```diff
@@ -357,15 +357,15 @@ func (e *Environment) createTestData() {
 	}
 }
 
-func (e *Environment) createClusters(count int) {
-	tempSkrs := make([]*envtest.Environment, 0)
+func (e *Environment) createClusters(skrCount int) {
+	tempSkrs := make([]*envtest.Environment, skrCount)
 	wg := &sync.WaitGroup{}
-	for i := 0; i <= count; i++ {
+	for i := 0; i <= skrCount; i++ {
```

Comment:
Why not:
`for i := 0; i < skrCount; i++ {
   creation of skrCount clusters
}
creation of one additional
`